posts/qrack_joins_uf.html

```diff
@@ -0,0 +1,169 @@
+
+        gtag('config', 'UA-19932157-4');
+    </script>
+    <title>Qrack and PyQrack join the Unitary Fund Organization</title>
+</head>
+
+<body>
+    <div class="layout">
+        <main id="content" class="content">
+            <section class="heavy">
+                <div class="container">
+                    <div class="col-3">
+                        <div class="hero">
+                            <a href="/"><img src="/logos/logov3.svg" alt="Unitary Fund" /></a>
+                            <p class="tagline"><b>Because evolution is
+                                    <a href="https://en.wikipedia.org/wiki/Unitarity_(physics)"
+                                        target="_blank">unitary</a></b>.</p>
+                            <button id="mobile-menu-icon" class="hamburger hamburger--squeeze" type="button">
+                                <span class="hamburger-box">
+                                    <span class="hamburger-inner"></span>
+                                </span>
+                            </button>
+                        </div>
+                        <section class="light">
+
+
+                            <right>
+                                <h2 class="leading-arrow">Qrack and PyQrack join the Unitary Fund Organization</h2>
+                                <h3></h3>
+                            </right>
+                            2022-07-21
+                            <br> Daniel Strano
+
+
+                            <div>
+                                <p>We announce with great pleasure and excitement that the quantum computer simulator libraries <a href="https://github.com/unitaryfund/qrack/">vm6502q/qrack</a> and <a href="https://github.com/unitaryfund/pyqrack/">vm6502q/pyqrack</a> have officially joined the Unitary Fund GitHub organization of open source quantum computing repositories! Get started with Qrack from the docs <a href="https://qrack.readthedocs.io/en/latest/">here</a>.</p>
+                                <p>Started by Daniel Strano (technical staff member of Unitary Fund) and Benn Bollay (CTO of Fusebit) in 2017 and winner of a Unitary Fund microgrant in 2019, Qrack is a quantum computer gate model simulator written in C++11. Qrack is intended to provide maximum performance and minimum memory footprint for quantum computing workloads run on “classical” computer hardware. PyQrack is Qrack’s dependency-free pure Python “ctypes” wrapper, to expose Qrack shared library binaries directly for just-in-time (JIT) execution via a Python interpreter. Plugins and providers are available for Qiskit, XACC, and even the Unity video game engine, with plans to expand plugin support to virtually every major quantum open source framework in the global ecosystem. Supported platforms include all available OEM combinations of x86_64, x86, ARMv7, and ARM64 instruction sets with Linux, Windows, Mac, iOS, and Android operating systems, as well as WebAssembly (Wasm).</p>
```

Comment:
1. why `"classical"`?
2. maybe "...dependency-free Python "ctypes wrapper..." (remove pure)
3. Can you add links to Qiskit, XACC, and maybe unity?
4. What is OEM?